app/org/sagebionetworks/bridge/stormpath/StormpathAccount.java

```diff
@@ -133,12 +155,16 @@ public void setHealthId(String healthId) {
         encryptTo(healthIdKey, healthId);
     };
     @Override
-    public ConsentSignature getConsentSignature() {
-        return decryptJSONFrom(consentSignatureKey, ConsentSignature.class);
+    public ConsentSignature getActiveConsentSignature() {
+        if (!signatures.isEmpty()) {
+            ConsentSignature signature = signatures.get(signatures.size()-1);
+            return (signature.getWithdrewOn() == null) ? signature : null;
+        }
+        return null;
```

Comment:
Minor: In Java 8, interfaces can have default implementations. Since getActiveConsentSignature() isn't dependent on the Stormpath implementation, could we move this to the interface?